samples/TagHelperSample.Web/Controllers/HomeController.cs

```diff
@@ -33,7 +33,7 @@ public IActionResult Index()
         // GET: /Home/Create
         public IActionResult Create()
         {
-            ViewBag.Items = _items;
+            ViewData["Items"] = _items;
```

Comment:
We should keep `ViewBag` around to for issues such as https://github.com/aspnet/Mvc/issues/3991. No reason to change it here.